### PR TITLE
Look for dependency versions on separate dependency indexes

### PR DESCRIPTION
**Pull request by fridex**
## Related Issues and Dependencies

Related: https://github.com/thoth-station/solver/issues/5111
Related: https://github.com/thoth-station/graph-refresh-job/issues/669
Related: https://github.com/thoth-station/thoth-application/issues/2272

## This introduces a breaking change

- [x] No

## Description

Pass all registered indexes when scheduling solver workflow.

### CODE REVIEW

File: thoth/solver/cli.py [hunk 1]
```diff
@@ -81,6 +86,16 @@ def cli(ctx=None, verbose=0):
     default="https://pypi.org/simple",
     help="A comma separated list of Python indexes to be used when resolving version ranges.",
 )
+@click.option(
+    "--dependency-index",
```
Comment (mayaCostantini) [inline thoth/solver/cli.py:90]: Should it be
```suggestion
    "--dependency-indexes",
```
If we have a list of indexes?

**Review by mayaCostantini — COMMENTED** [review]
Just one small doubt, otherwise lgtm :100: :+1:

File: thoth/solver/cli.py (hunk 1 above)
Comment (fridex) [reply thoth/solver/cli.py:90]: I see your point, I was unsure about it as well. I followed the already existing ``--index`` option in this case.

Comment (mayaCostantini) [reply thoth/solver/cli.py:90]: Makes sense, we can leave it as it is then :+1:

**Pull request merged**